react/features/chat/components/web/ChatInput.js

```diff
@@ -55,12 +60,7 @@ type State = {
     /**
      * Whether or not the smiley selector is visible.
      */
-    showSmileysPanel: boolean,
-
-    /**
-     * Window size
-     */
-    width: number
```

Comment:
this is needed from the upstream i think

Reply:
> this is needed from the upstream i think

Could you recheck, please? I deleted only duplicates, in props validation and in state we also had the same.